test/steps/mappers/create_maven_options.go

```diff
@@ -26,14 +26,13 @@ const (
 	mavenProfileKey = "profile"
 	mavenOptionKey  = "option"
 	mavenNativeKey  = "native"
-
-	nativeProfile = "native"
 )
 
 // MavenCommandConfig contains configuration for Maven Command execution
 type MavenCommandConfig struct {
 	Profiles []string
 	Options  []string
+	Native   bool
```

Comment:
Switch to independent value because `native` is a package type and not a profile (it is for our example but we may change that later)
Also this will make more coherent image names